### PR TITLE
ColorBox: fix providing information after change value by Arrows press and on update by Enter click

### PR DESCRIPTION
**Pull request by Krijovnick**

### CODE REVIEW

File: testing/tests/DevExpress.ui.widgets.editors/colorBox.markup.tests.js [hunk 1]
```diff
@@ -35,12 +35,20 @@ QUnit.module('ColorBox', {
     });
 
     QUnit.test('Render color input', function(assert) {
-        const $colorBox = showColorBox.call(this); const $input = $colorBox.find('.' + COLOR_BOX_INPUT_CLASS);
+        const $colorBox = showColorBox.call(this);
+        const $input = $colorBox.find('.' + COLOR_BOX_INPUT_CLASS);
```
Comment (marker-dao) [inline testing/tests/DevExpress.ui.widgets.editors/colorBox.markup.tests.js]:
```suggestion
        const $input = $colorBox.find(`.${COLOR_BOX_INPUT_CLASS}`);
```

Comment (AlexanderMoiseev) [inline testing/tests/DevExpress.ui.widgets.editors/colorBox.markup.tests.js]: [minor] - just to stay consistent
```suggestion
        const $input = $colorBox.find(`.${COLOR_BOX_INPUT_CLASS}`);
```

Comment (Krijovnick) [reply testing/tests/DevExpress.ui.widgets.editors/colorBox.markup.tests.js]: It is not my code, I just move it to new line

File: js/ui/color_box/color_box.js [hunk 2]
```diff
@@ -319,6 +319,9 @@ const ColorBox = DropDownEditor.inherit({
 
         this.option('text', text);
 
+        const $input = this._input();
+        this.setAria('live', 'polite', $input);
```
Comment (AlexanderMoiseev) [inline js/ui/color_box/color_box.js]: NVDA reader doesn't read the value on enter key pressed when colorBox popup is closed. But i'n not sure this is a problem

Comment (marker-dao) [reply js/ui/color_box/color_box.js]: My version of nvda reads this

**Review by ksercs — APPROVED** [review]

**Review by AlexanderMoiseev — APPROVED** [review]

**Review by marker-dao — APPROVED** [review]

**Pull request merged**